modules/library/coverage/src/test/java/org/geotools/image/ImageWorkerTest.java

```diff
@@ -924,6 +925,7 @@ public void testOpacityAlphaRGBDirect() {
     }
     
     @Test
+    @Ignore ("Due to LibCMS Bug")
```

Comment:
This actually only happens in openjdk under some linux distros (not all, 12.04 for example is not affected), maybe make this as "assume" test, checking if the jdk is a openjdk?